pkg/cmd/image/remove.go

```diff
@@ -77,8 +77,8 @@ func Remove(ctx context.Context, client *containerd.Client, args []string, optio
 				return nil
 			}
 
-			if cid, ok := runningImages[found.Image.Name]; ok {
-				return fmt.Errorf("conflict: unable to delete %s (cannot be forced) - image is being used by running container %s", found.Req, cid)
+			if cid, ok := runningImages[found.Image.Name]; !options.Force && ok {
+				return fmt.Errorf("conflict: unable to delete %s - image is being used by running container %s", found.Req, cid)
```

Comment:
I saw the same behavior, but the docs seem to be same as the behavior in this PR

https://docs.docker.com/reference/cli/docker/image/rm/

From the docs

docker images

REPOSITORY                TAG                 IMAGE ID            CREATED             SIZE
test1                     latest              fd484f19954f        23 seconds ago      7 B (virtual 4.964 MB)
test                      latest              fd484f19954f        23 seconds ago      7 B (virtual 4.964 MB)
test2                     latest              fd484f19954f        23 seconds ago      7 B (virtual 4.964 MB)

docker rmi -f fd484f19954f

Untagged: test1:latest
Untagged: test:latest
Untagged: test2:latest
Deleted: fd484f19954f4920da7ff372b5067f5b7ddb2fd3830cecd17b96ea9e286ba5b8

Reply:
> since docker will have to pull the digest again in both the scenarios.

can you explain this a bit?

In docker's case, it doesn't actually delete the image right? so if I re-pull the image using the same tag, then it just associate the tag with the `<none>:<none>` image left previously?

```shell
 ~/  ➜ d run -it -d ubuntu:22.04 sleep 30
8f8f03e6c804a70fe6fba790b0d894b97389ba2e94283c37282aa9079fdc7976
 ~/  ➜ d rmi -f ubuntu:22.04
Untagged: ubuntu:22.04
 ~/  ➜ d images
REPOSITORY   TAG       IMAGE ID       CREATED       SIZE
<none>       <none>    0e5e4a57c249   6 weeks ago   106MB
 ~/  ➜ d pull ubuntu:22.04
22.04: Pulling from library/ubuntu
Digest: sha256:0e5e4a57c2499249aafc3b40fcd541e9a456aab7296681a3994d631587203f97
Status: Downloaded newer image for ubuntu:22.04
docker.io/library/ubuntu:22.04
 ~/  ➜ d images
REPOSITORY   TAG       IMAGE ID       CREATED       SIZE
ubuntu       22.04     0e5e4a57c249   6 weeks ago   106MB
```

Reply:
Notice the original comments:

> 			// If an image is associated with a running/paused containers, `docker rmi -f imageName`
>			// untags `imageName` (left a `<none>` image) without deletion; `docker rmi -rf imageID` fails.
>			// In both cases, `nerdctl rmi -f` will fail.

what's the new behavior of `nerdctl rmi -f imageName` and `nerdctl rmi -f imageID`?

Reply:
>what's the new behavior of nerdctl rmi -f imageName and nerdctl rmi -f imageID?
It does delete, so we need to skip the untag part, I will check, what can be done.

>can you explain this a bit?

the image ids for the same image are different in docker and nerdctl.
For docker, it uses the config-sha256, for nerdctl it uses digest / index-sha256